bin/git-utimes

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+#
+# Change files modification time to their last commit date
+#
+# Copyright (c) 2020 Vitaly Chikunov <vt@altlinux.org>.
+# SPDX-License-Identifier: MIT
+
+if [ "$1" = --touch ]; then
+  # Internal use option only just to parallelize things.
+  shift
+  for f; do
```

Comment:
Is the for loop required?

Reply:
`for` is to support multiple arguments. Why not? This should reduce number of `exec`s. Are you against of it?

Reply:
Correction: s/execs/forks/

Reply:
I got it.